app/app-controllers/searchBarController.js

```diff
@@ -41,36 +42,40 @@ function(
 
     // You must manually call this for the search bar to be visible
     SearchBarController.prototype.addToLayout = function() {
-        this.layout.addTopView(this.viewPlugin, {height: 52});
+        this.layout.addTopView(this.viewPlugin);
         this.hide({animated: false}); // don't show it by default
     };
 
     SearchBarController.prototype._registerEvents = function() {
         var self = this;
-
         // Users of SearchBarController can also hook this event
         // to instruct the desired view to act on the search being performed
         // The search terms are expected to be in a string found in
         // params['searchTerms']
-        this.viewPlugin.on('search:submitted', function(params) {
-            self.hide({animated: true});
+        this.searchBar.on('search:submitted', function(params) {
+            self.hide();
         });
 
-        this.viewPlugin.on('search:cancelled', function(params) {
-            self.hide({animated: true});
+        this.searchBar.on('search:cancelled', function(params) {
+            self.hide();
         });
     };
 
     SearchBarController.prototype.show = function(options) {
+        options = options || (Astro.isRunningInIOSApp() ? {animated: true} : {animated: false});
```

Comment:
The `options` here are being used for `AnchoredLayoutPlugin`'s `showTopView`/`hideTopView` though, not sure what we will affect if we add default params for it now?

I turned off animations on Android because it looked kinda weird (in Android the current webview is replaced by a white screen) and with animations on, the entire screen wipes. I wouldn't be against turning the animations back on instead if we all agree 😜